src/vite-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="vite-plugin-svgr/client" />
+/// <reference types="vite/client" />
```

Comment:
What is the purpose of this file ?

Reply:
For TypeScript.
Was already existing under an other name.
https://github.com/gridsuite/gridstudy-app/pull/1919/files#diff-433083e9966b3a808a0b2dce3e47b0b74152c61cb7d5b6a4ad8aad2592dacd81